civiform/civiform
# Add Primary Applicant Info toggles to the question edit view

### Description

This adds the Primary Applicant Information toggles to the question edit view.  If the universal toggle is not set, it shows an alert saying the PAI tag can't be set because it is not a universal question.  If the universal toggle is set, but the tag is already set on another question, it shows an alert saying so.  Upon question creation, when an enumerator is set, the PAI section is hidden.  All of this is done dynamically via a bit of JS code.

This also changes the name of the feature flag, since it was created before we finalized the name of this feature.  I promise it won't change again :)

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


#### New Features


### Instructions for manual testing

1. Enable the PRIMARY_APPLICANT_INFO_QUESTIONS_ENABLED feature flag.
2. Ensure the alerts and toggles are shown and hidden according to the description.
3. Ensure the state of the toggles persist after saving and publishing.
4. Optionally, check the questions table in the database to verify the appropriate QuestionTag was added to the list for the question.

### Issue(s) this completes

https://github.com/civiform/civiform/issues/6397
https://github.com/civiform/civiform/issues/6398
